types/typeProvider.d.ts

```diff
@@ -0,0 +1,10 @@
+export interface FastifyTypeProvider {
```

Comment:
Can you rename this file to `type-provider.d.ts`? just to follow the pattern in this folder.

Reply:
Renamed to `type-provider.d.ts` as per request.